vro-types/o11n-plugin-vc/index.d.ts

```diff
@@ -15833,6 +15833,37 @@ declare class VcVirtualMachineBackupEventInfo {
 	constructor(eventType: string, code: number, message: string);
 }
 
+/**
+ * Provice information about storage
+ */
+declare class VcStorageQueryManager {
+  readonly vimType: string;
+  readonly sdkConnection: VcSdkConnection;
+  readonly moref: VcManagedObjectReference;
+  readonly vimId: string;
+  readonly sdkId: string;
+  readonly vimHost: VcSdkConnection;
+
```

Comment:
I see another function in the API - _getRef, maybe it is worth adding it as well
![image](https://github.com/user-attachments/assets/f03561b9-3b92-454d-aa70-f5fc11851357)